inspirehep/modules/records/jsonschemas/records/elements/reference.json

```diff
@@ -0,0 +1,163 @@
+{
+    "description": "An article or thesis or book or...",
+    "title": "Reference",
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "texkey": {
+            "type": "string",
+            "description": "INSPIRE Provided texkey."
+        },
+        "number": {
+            "type": "integer",
+            "description": "Position within the reference list of a record (the number you find in brackets on the PDF)."
+        },
+        "raw_reference": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "value": {
+                        "type": "string"
+                    },
+                    "serialization": {
+                        "type": "string",
+                        "description": "E.g. refextract, text, JATS, Elsevier, BibTeX..."
```

Comment:
Elsevier references come directly in a json format, while others come in text. I think the examples are not particularly good. We could actually have both source and format (`text`, `json`, `bibtex`, `xml`)